hstream/src/HStream/Server/Handler.hs

```diff
@@ -116,6 +115,11 @@ genQueryResultResponse :: V.Vector Struct -> CommandQueryResponse
 genQueryResultResponse = CommandQueryResponse .
   Just . CommandQueryResponseKindResultSet . CommandQueryResultSet
 
+batchAppend :: LDClient -> TL.Text -> [Bytes] -> IO (Either SomeException AppendCompletion)
+batchAppend client streamName payloads = do
+  logId <- getCLogIDByStreamName client $ transToStreamName $ TL.toStrict streamName
+  try $ appendBatch client logId payloads CompressionNone Nothing
```

Comment:
Why use `CompressionNone` instead of `CompressionLZ4`?

- The default compression in CPP code is LZ4
- Should we read this option from the config file instead of hard coding?

Reply:
Get, I'll add an option in `parseConfig` and set the default option to LZ4